snow/engine/common/test_config.go

```diff
@@ -37,7 +37,6 @@ func DefaultConfigTest() Config {
 		Bootstrapable:                  &BootstrapableTest{},
 		BootstrapTracker:               bootstrapTracker,
 		Timer:                          &TimerTest{},
-		AncestorsMaxContainersSent:     2000,
```

Comment:
if we got for including GetAncestors config in a single struct, we should include AncestorsMaxContainerReceived as well